.devops/repo/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "repo"
+version = "0.1.0"
+edition = "2021"
+authors=["Fuel Labs <contact@fuel.sh>"]
```

```suggestion
authors = ["Fuel Labs <contact@fuel.sh>"]
publish = false
```

Spacing, and since this package probably isn't meant to be published, let's add `publish = false` here.